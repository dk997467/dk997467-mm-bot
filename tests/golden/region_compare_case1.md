Region Canary Comparison

| region | net_bps | order_age_p95_ms | fill_rate | taker_share_pct |
|--------|---------|------------------|-----------|------------------|
| ap-sg | 2.850000 | 395.000000 | 0.600000 | 12.500000 |
| eu-west | 2.650000 | 340.000000 | 0.590000 | 12.750000 |
| us-east | 2.450000 | 380.000000 | 0.580000 | 14.000000 |

| window | net_bps | order_age_p95_ms | fill_rate | taker_share_pct |
|--------|---------|------------------|-----------|------------------|
| 00:00-02:00 | 2.800000 | 300.000000 | 0.625000 | 11.750000 |
| 02:00-04:00 | 2.350000 | 380.000000 | 0.555000 | 15.250000 |

Winner: us-east @ 00:00-02:00


