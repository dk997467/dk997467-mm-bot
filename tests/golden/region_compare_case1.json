{"regions":{"ap-sg":{"fill_rate":0.6,"net_bps":2.85,"order_age_p95_ms":400.0,"taker_share_pct":12.5},"eu-west":{"fill_rate":0.59,"net_bps":2.6500000000000004,"order_age_p95_ms":340.0,"taker_share_pct":12.75},"us-east":{"fill_rate":0.5800000000000001,"net_bps":2.45,"order_age_p95_ms":380.0,"taker_share_pct":14.0}},"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"},"windows":{"00:00-02:00":{"fill_rate":0.625,"net_bps":2.8,"order_age_p95_ms":300.0,"taker_share_pct":11.75},"02:00-04:00":{"fill_rate":0.555,"net_bps":2.3499999999999996,"order_age_p95_ms":380.0,"taker_share_pct":15.25}},"winner":{"region":"us-east","window":"00:00-02:00"}}
